templates/ingress-gateways-deployment.yaml

```diff
@@ -417,13 +419,17 @@ spec:
               value: http://$(HOST_IP):8500
             {{- end }}
           command:
-            - consul-k8s
-            - consul-sidecar
-            - -service-config=/consul/service/service.hcl
-            - -consul-binary=/consul-bin/consul
-            {{- if $root.Values.global.acls.manageSystemACLs }}
-            - -token-file=/consul/service/acl-token
-            {{- end }}
+            - "/bin/sh"
+            - "-ec"
```

Comment:
should we leave this as before? One thing with running in `/bin/sh` is you don't get passed signals when the pod stops. To get around this we could use `exec` but maybe in a sep PR we could standardize

Reply:
Good call, reverting!